pkg/controllers/multiclusterservice/endpointslice_collect_controller.go

```diff
@@ -84,7 +84,7 @@ const EndpointSliceCollectControllerName = "endpointslice-collect-controller"
 
 // Reconcile performs a full reconciliation for the object referred to by the Request.
 func (c *EndpointSliceCollectController) Reconcile(ctx context.Context, req controllerruntime.Request) (controllerruntime.Result, error) {
-	klog.V(4).Infof("Reconciling Work %s", req.NamespacedName.String())
+	klog.V(4).InfoS("Reconciling Work", "namespacedName", req.NamespacedName.String())
```

Comment:
Here, can you add keys for both the namespace and name like done in the log calls below?